servers/eap8.0/src/main/java/org/jboss/migration/eap/EAPXPServer8_0.java

```diff
@@ -36,15 +36,21 @@ public class EAPXPServer8_0 extends WildFlyServer10 {
 
     public static final Extensions EXTENSIONS = Extensions.builder()
             .extensions(EAPServer8_0.EXTENSIONS)
+            .extension(JBossExtensions.MICROMETER)
             .extension(JBossExtensions.MICROPROFILE_CONFIG_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_FAULT_TOLERANCE_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_HEALTH_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_JWT_SMALLRYE)
+            // workaround for issue with migrate ops
             .extension(JBossExtensions.MICROPROFILE_METRICS_SMALLRYE)
+            .extension(JBossExtensions.MICROPROFILE_LRA_COORDINATOR)
+            .extension(JBossExtensions.MICROPROFILE_LRA_PARTICIPANT)
             .extension(JBossExtensions.MICROPROFILE_OPENAPI_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_OPENTRACING_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_REACTIVE_MESSAGING_SMALLRYE)
             .extension(JBossExtensions.MICROPROFILE_REACTIVE_STREAMS_OPERATORS_SMALLRYE)
+            .extension(JBossExtensions.MICROPROFILE_TELEMETRY)
```

Comment:
I guess we could add it if it's filename matches one of the default microprofile configurations (standalone-microprofile.xml, standalone-microprofile-ha.xml)? But I dunno if this is a must add or not, if it's optional I think we should not add it.

@kabir @bstansberry @jasondlee ^^ WDYT?

Reply:
no policy, I have been trying to use common sense and add if it's needed to work (and as expected), skip it otherwise

Reply:
My main concern is about functionality of OpenTelemetry which is added by the migration operation of the MP OpenTracing subsystem. I don't understand the relationship between `microprofile.telemetry` and `opentelemetry` subsystems. `opentelemetry` is added instead of `microprofile.opentracing-smallrye` but `microprofile.telemetry` not.

Reply:
We will then not add the subsystem, if there is any post release issue then we consider it as bug